src/utils/chain.ts

```diff
@@ -140,7 +140,7 @@ export const nativeAssetPerChainId: { [chainId: number]: TokenListToken } = {
     name: 'Optimism',
     symbol: 'ETH',
     decimals: 18,
-    logoURI: 'https://public.etherspot.io/buidler/chain_logos/avalanche.svg',
+    logoURI: 'https://public.etherspot.io/buidler/chain_logos/ethereum.png',
```

Comment:
@arddluma why are we showing ethereum logo for Optimism and Arbitrum?

Reply:
> @arddluma why are we showing ethereum logo for Optimism and Arbitrum?
This is for native tokens. And on Opt and Arbi both have Ether as native token.